Mathlib/CategoryTheory/Comma/Over.lean

```diff
@@ -657,4 +657,140 @@ lemma toUnder_comp_forget (F : S ⥤ T) (X : T) (f : (Y : S) → X ⟶ F.obj Y)
 
 end Functor
 
+namespace StructuredArrow
+
+/-- A functor from the structured arrow category on the projection functor for any structured
+arrow category. -/
+@[simps!]
+def ofStructuredArrowProjFunctorFrom {D : Type u₂} [Category.{v₂} D] (F : D ⥤ T) (Y : T) (X : D) :
```

Comment:
I would prefer names like:
```suggestion
def ofStructuredArrowProjEquivalence.functor {D : Type u₂} [Category.{v₂} D] (F : D ⥤ T) (Y : T) (X : D) :
```
(Also, as `D` is used multiple times, it could be declared as variable by the beginning of the file.)

Reply:
What would you call the inverse functors?

Reply:
`ofStructuredArrowProjEquivalence.inverse`!